config/config.go

```diff
@@ -87,6 +87,28 @@ type Proxy struct {
 
 		// Version of the Kafka cluster. Supported versions are 0.10.2.1 - 2.0.0
 		Version KafkaVersion
+
+		// Optionally use TLS when connecting to Kafka. This must be enabled
+		// for following options to be used.
+		TLSEnabled bool `yaml:"tls"`
+
+		// The path to the CA certificate (PEM)
+		CACertFile string `yaml:"ca_certificate_path"`
```

Comment:
Let's be consistent and call it `file` in the config too. The same applies to the two following parameters. Respectively `testdata/kafka-tls.yaml` needs to be updated.